Dockerfile

```diff
@@ -1,15 +1,28 @@
-FROM ipython/scipystack
+FROM ubuntu:14.04
```

Comment:
Yeah, up until the Docker Hub outage happened this morning. Then I switched back to an image I already had on my system which of course was ubuntu:14.04.

The ubuntu image is a lot smaller than it used to be now I just noticed.